packages/webapp/src/components/Form/InputBase/InputBaseField/styles.module.scss

```diff
@@ -13,14 +13,14 @@
  *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
  */
 
-.input input {
+.inputWrapper .input {
   all: unset;
```

Comment:
Added a className to `input` so that `all: unset;` will not be applied to the input inside ReactSelect.